OG_Format.adoc

```diff
@@ -759,7 +759,7 @@ vocabulary = "";
 
 <PARAM>_QC:RTQC_methodology_doi = “”;
 
- |mandatory
+ |higly-desirable
```

Comment:
highly desirable is not hyphenated elsewhere in the document. I would suggest removing the hyphen for consistency